lib/rouge/lexers/make.rb

```diff
@@ -11,9 +11,10 @@ class Make < RegexLexer
       filenames '*.make', 'Makefile', 'makefile', 'Makefile.*', 'GNUmakefile'
       mimetypes 'text/x-makefile'
 
-      bsd_special = %w(
-        include undef error warning if else elif endif for endfor
-      )
+      # TODO: Add support for special keywords
+      # bsd_special = %w(
+      #   include undef error warning if else elif endif for endfor
+      # )
```

Comment:
oh wow this was an unused variable...?

Reply:
Yeah. It's perhaps because [the way](https://bitbucket.org/birkenfeld/pygments-main/src/default/pygments/lexers/make.py) that Pygments uses it is a bit unusual. Or at least was to me. I tried in an earlier version of this commit to incorporate it in but couldn't work out quickly how to do it so left it for later.